Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -1,319 +1,112 @@
+  display: none;
 }
 
+div.tabbedMainMenu span.tabStripTab,
+div.tabbedMainMenu span.tabStripTabSelected,
 div.tabbedSubMenu span.tabStripTab,
 div.tabbedSubMenu span.tabStripTabSelected
 {
-  border-right: solid 1px #AAAAAA;
-
-  padding-top: 0.1em;
-  padding-left: 0.2em;
-  padding-right: 0.2em;
-
-  min-height: 1.4em;
-  min-height: calc(16px + 0.2em);
-}
-
-div.tabbedSubMenu div.tabStripTabsPane span.tabStripTabWrapper
-{
-  /* Compensates for 1px offset due to rounding errors when calculating the height in Internet Explorer. 
-    This might only be an issue if the TabbedMenu is displayed at the top of the page and manifests itself by jumping when moving the keyboard focus out of the TabbedMenu. */
-  padding-top: 0.1em;
-  padding-bottom: 0.1em;
-}
-
-div.tabbedSubMenu span.tabStripTab a,
-div.tabbedSubMenu span.tabStripTabSelected a
-{
-  display: block;
-  position: relative;
-  min-height: 16px;
-}
-
-div.tabbedSubMenu span.tabStripTab a span.anchorBody,
-div.tabbedSubMenu span.tabStripTabSelected a span.anchorBody
-{
-  /* Compensates the margin-top: -0.1em for the img-tag */
-  padding-top: 0.1em;
-}
-
-div.tabbedSubMenu span.tabStripTab a span.anchorBody > img,
-div.tabbedSubMenu span.tabStripTabSelected a span.anchorBody > img
-{
-  position: relative;
-  top: -0.1em;
-  vertical-align: middle;
-}
-
-div.tabbedSubMenu span.tabStripTab a span.anchorBody > span,
-div.tabbedSubMenu span.tabStripTabSelected a span.anchorBody > span
-{
-}
-
-div.tabbedSubMenu span.tabStripTab a
-{
+  padding: 0;
   border: none;
-  background: none;
-  background-color: White;
-  color: Black;
-  text-decoration: none;
+  background-color: transparent;
+  align-items: center;
+  min-height: calc(var(--remotion-themed-spacing) * 7);
 }
 
-div.tabbedSubMenu span.tabStripTab a:focus,
-div.tabbedSubMenu span.tabStripTabSelected a:focus
+div.tabbedMainMenu span.tabStripTab:not(.disabled):hover,
+div.tabbedSubMenu span.tabStripTab:not(.disabled):hover
 {
-  outline: dotted black 1px;
-  outline: auto 5px -webkit-focus-ring-color;
+  background-color: transparent;
 }
 
-div.tabbedSubMenu span.tabStripTab a:focus,
-div.tabbedSubMenu span.tabStripTab a:hover,
-div.tabbedSubMenu span.tabStripTab a:active,
-div.tabbedSubMenu span.tabStripTabSelected a:focus,
-div.tabbedSubMenu span.tabStripTabSelected a:hover,
-div.tabbedSubMenu span.tabStripTabSelected a:active
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled) a,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled) a,
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a:hover,
+div.tabbedMainMenu span.tabStripTab:not(.disabled) a:focus,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled) a:focus,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a:hover,
+div.tabbedSubMenu span.tabStripTab:not(.disabled) a:focus,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled) a:focus
```

Comment:
I did not try this, but there seems to be a lot of stuff here, that could also be moved up to the neutral block in line 49/52 since it is always valid and isn't overridden via the tabstrip-style (I think, please check and simplify as possible).